### PR TITLE
Disable markdown link check and enable presubmit test fail fast

### PR DESCRIPTION
**Pull request by navidshaikh**
sets `DISABLE_MD_LINK_CHECK=1` and `PRESUBMIT_TEST_FAIL_FAST=1`

### CODE REVIEW

**Comment by navidshaikh** [comment]
See test https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_client/685/pull-knative-client-build-tests/1232328125652668416 , while the [link](https://docs.openshift.com/container-platform/4.1/cli_reference/administrator-cli-commands.html#create-kubeconfig) works.
This is causing the build tests to fail for unrelated changes, https://github.com/knative/client/pull/676#issuecomment-590925404

**Comment by navidshaikh** [comment]
looks like the linting tool used in test-infra doesn't support `#` in the URLs
```
➜  client git:(pr/fix-md-linting-err) liche docs/README.md 
docs/README.md
	ERROR	https://docs.openshift.com/container-platform/4.1/cli_reference/administrator-cli-commands.html#create-kubeconfig
		Bad Request (HTTP error 400)
```
/cc: @adrcunha

**Comment by adrcunha** [comment]
/cc @chizhg 

Please note that I am not part of Knative anymore.

**Comment by navidshaikh** [comment]
/test pull-knative-client-integration-tests
```
    cli.go:70: ERROR: Operation cannot be fulfilled on apiserversources.sources.eventing.knative.dev "testapisource3": the object has been modified; please apply your changes to the latest version and try again
```

**Comment by navidshaikh** [comment]
/retest

async revision delete issue

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -32,6 +32,8 @@
 # Serving / Eventing of this specific version.
 
 export DISABLE_MD_LINTING=1
+export DISABLE_MD_LINK_CHECK=1
```
Comment (chizhg) [inline test/presubmit-tests.sh]: Can you add a TODO here? If we have a chance to fix that issue in liche in the future, we can reenable it.

Comment (navidshaikh) [reply test/presubmit-tests.sh]: Sorry! forgot to add TODO, I'd filed https://github.com/raviqqe/liche/issues/33

Comment (chizhg) [reply test/presubmit-tests.sh]: No worries. Thanks.

**Comment by navidshaikh** [comment]
/retest

ERROR: timeout waiting for pods to come up

**Pull request merged**